fvm/context.go

```diff
@@ -59,9 +60,18 @@ func newContext(ctx Context, opts ...Option) Context {
 const AccountKeyWeightThreshold = 1000
 
 const (
-	DefaultGasLimit                     = 100_000 // 100K
-	DefaultEventCollectionByteSizeLimit = 256_000 // 256KB
+	DefaultComputationLimit             = 100_000     // 100K
+	DefaultMemoryLimit                  = 100_000_000 // 100G
+	DefaultEventCollectionByteSizeLimit = 256_000     // 256KB
 	DefaultMaxNumOfTxRetries            = 3
+
+	// service transaction is a transaction have service account as payer
+	// and signed by service account, higher limits are allowed
+	// for service transactions to consider flexibility for network services
+	// such as governance transactions, etc.
+	ServiceTransactionComputationLimit             = 100_000_000_000 // 100B
```

Comment:
Did you mean 100T? Are we sure about that? 100G already seems very high
```suggestion
	ServiceTransactionComputationLimit             = 100_000_000_000 // 100T
```

Reply:
so we didn't have any limit before, this is only the case that if service account accidentally forgets to set a limit, and submit an always for loop, I was not sure what we should set for them, and didn't want to cause any disturbance to the reward payout and etc, @Kay-Zee any idea what we could do as a upperbound for service transactions?

Reply:
this seems to be a separate problem to be solved, move the logic outside of this PR.